<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>cflow-gui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/fork-8x.png</normaloff>:/images/images/fork-8x.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QListWidget" name="listWidgetFunctions"/>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEditRootPath"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelFunctions">
      <property name="text">
       <string>Selected functions to show:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAll">
        <property name="text">
         <string>&amp;All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNone">
        <property name="text">
         <string>&amp;None</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="fileSystemTreeView"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEditFilter">
      <property name="placeholderText">
       <string>Filter (regular expression)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPath"/>
   <addaction name="actionGenerate"/>
   <addaction name="actionShowGraph"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPath">
   <property name="icon">
    <iconset>
     <normalon>:/images/images/folder-2x.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Change Path</string>
   </property>
   <property name="toolTip">
    <string>Change Path</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionShowGraph">
   <property name="icon">
    <iconset>
     <normalon>:/images/images/eye-2x.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show graph</string>
   </property>
   <property name="toolTip">
    <string>Show Graph as PDF</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="icon">
    <iconset>
     <normalon>:/images/images/share-boxed-2x.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>Generate Graph</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
